test/meson.build

```diff
@@ -0,0 +1,13 @@
+test_dependencies = dependencies + dependency('cmocka')
+
+test_args = executable('test_args',
+                       ['test_args.c', '../args.c'],
+                       include_directories:'..',
+                       dependencies:test_dependencies)
+test('args', test_args, workdir:meson.source_root())
```

Comment:
I would suggest using `meson.project_source_root()` here, because source_root was deprecated in Meson 0.56.0

Meson will print deprecation warnings if you specify a `meson_version: '>=0.60.0'` kwarg to the project() function, which aligns with what is available on Jammy and Lunar anyway. (It will also print warnings if you use too new features in development, before you push those changes to CI and have it fail on older Ubuntu releases.)

Reply:
Done